### ISSUE TITLE
symptoms localized to pulp of the finger

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10717]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
Symptoms are mainly found on the fleshy mass on the volar (palmar) aspect of the digit, distal to the DIP joint

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Felon

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001199
- label: FInger pulp localization
- parents: [Spatial pattern (HP:0012836)](https://hpo.jax.org/app/browse/term/HP:0012836)
- pmid: 
- def: Applies to an abnormality whose distribution is limited to or predominantly affects the fleshy mass on the volar (palmar) aspect of the digit, distal to the distal interphalangeal (DIP) joint (known as the pulp of the finger).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10717
- synonyms: Symptoms localized to pulp of the finger(NONE)
